crawlers/ten_ago_crawler.py

```
@@ -0,0 +1,138 @@
+            except:
+                tag = driver.find_element(By.CSS_SELECTOR, "ul.breadcrumb > li:nth-child(2)")
+                category = tag.text.strip()
+
+            category = ' '.join(category.strip().split())
+            print(f"✅ [{category}] {url}")
+
+            if category == target_category:
+                print(f"🎯 [{category}] 기사 발견! → {url}")
+                return url
+
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)